File: 0020-valid-parentheses/brute-force/python/solution.py
class Solution:
    def isValid(self, s: str) -> bool:
        cur_str = s
        while "()" in cur_str or "{}" in cur_str or "[]" in cur_str:
            cur_str = cur_str.replace("()", "")
            cur_str = cur_str.replace("{}", "")
            cur_str = cur_str.replace("[]", "")
        return cur_str == ""

File: 0020-valid-parentheses/brute-force/java/solution.java
class Solution {
    public boolean isValid(String s) {
        String curStr = s;
        int prevLength;

        do {
            prevLength = curStr.length();
            curStr = curStr.replace("()", "")
                           .replace("{}", "")
                           .replace("[]", "");
        } while (curStr.length() < prevLength);

        return curStr.isEmpty();
    }
}
